pkg/k8s/apis/cilium.io/v2alpha1/bgp_peer_types.go

```diff
@@ -124,14 +124,17 @@ type CiliumBGPFamilyWithAdverts struct {
 
 // CiliumBGPTransport defines the BGP transport parameters for the peer.
 type CiliumBGPTransport struct {
+	// Deprecated
```

Comment:
@julianwiedmann I wanted to continue the discussion regarding - whether we should do a version bump. 

I am not entirely sure, but in theory, we might need to bump the version. As Cilium is upgraded we want this schema to be reflected instead of the old one.

Generally, what is the criteria for bumping the version in stable branches? Should going from "1.29.11" -> "1.29.12" acceptable?

Reply:
yep, that's what I would expect as the the next version!